lib/engine.py

```diff
@@ -173,6 +141,30 @@ def source_gen():
         print ("\n")
         self.connection.commit()
 
+    def get_ct_data(self, lines, real_lines):
+        """Creates cross tab data"""
```

Comment:
As noted about I think that `real_lines = []` should be the first line of this function rather than doing it outside the function and passing `real_lines` in.